docs/sources/auth/google.md

```diff
@@ -29,19 +29,24 @@ First, you need to create a Google OAuth Client:
 Specify the Client ID and Secret in the [Grafana configuration file]({{< relref "../administration/configuration.md#config-file-locations" >}}). For example:
 
 ```bash
+#################################### Google Auth #########################
 [auth.google]
 enabled = true
-client_id = CLIENT_ID
-client_secret = CLIENT_SECRET
+allow_sign_up = true
+client_id = some_client_id
+client_secret =
 scopes = https://www.googleapis.com/auth/userinfo.profile https://www.googleapis.com/auth/userinfo.email
 auth_url = https://accounts.google.com/o/oauth2/auth
 token_url = https://accounts.google.com/o/oauth2/token
+api_url = https://www.googleapis.com/oauth2/v1/userinfo
 allowed_domains = mycompany.com mycompany.org
-allow_sign_up = true
+hosted_domain =
 ```
 
+For more information on these properties, refer to [Using OAuth 2.0 to Access Google APIs](https://developers.google.com/identity/protocols/oauth2/openid-connect).
```

Comment:
This is maybe slightly misleading. There is an oauth standard that all the oauth flows implement and then there are a few extra fields that are only for Google OAuth. Some of the properties are for Grafana logic and not part of the OAuth standard - `allowed_domains` and `allow_signup`.

Reply:
Let me open a separate PR.

> @achatterjee-grafana this whole page is a bit confusing as some of the Google OAuth-specific properties have no explanation and some of the properties are documented on the generic oauth page(https://grafana.com/docs/grafana/latest/auth/generic-oauth/).
> 
> Is it worth restructuring this page and finding a different format to properly explain what each property is? What do you think?

Yes, I will open a new PR and work on reorganizing the content.